x-pack/plugins/security_solution/server/endpoint/utils/audit_log_helpers.ts

```diff
@@ -32,7 +32,8 @@ import {
 import { doesLogsEndpointActionsIndexExist } from '../utils';
 
 const actionsIndices = [AGENT_ACTIONS_INDEX, ENDPOINT_ACTIONS_INDEX];
-const responseIndices = [AGENT_ACTIONS_RESULTS_INDEX, ENDPOINT_ACTION_RESPONSES_INDEX];
+// search all responses indices irrelevant of namespace
+const responseIndices = [AGENT_ACTIONS_RESULTS_INDEX, ENDPOINT_ACTION_RESPONSES_INDEX_PATTERN];
 export const logsEndpointActionsRegex = new RegExp(`(^\.ds-\.logs-endpoint\.actions-default-).+`);
 export const logsEndpointResponsesRegex = new RegExp(
   `(^\.ds-\.logs-endpoint\.action\.responses-default-).+`
```

Comment:
Do we need to adjust this pattern?

Reply:
Yes, I think so, along with the pattern for actions index.